frontend/src/pages/add-blog.tsx

```diff
@@ -89,95 +89,99 @@ function AddBlog() {
   };
   const navigate = useNavigate();
   return (
-    <div className="font-[Poppins] p-4 px-16 bg-white ">
-      <div className="flex justify-start items-center mb-4">
-        <div className=" text-black cursor-pointer w-fit  text-base md:text-lg lg:text-2xl">
-          <FiArrowLeft onClick={() => navigate(-1)} />
+    <div className="dark:dark w-full min-h-screen">
+      <div className="font-[Poppins] p-4 px-16 bg-white dark:dark ">
+        <div className="flex justify-start items-center mb-4">
+          <div className=" text-black cursor-pointer w-fit  text-base md:text-lg lg:text-2xl dark:text-white">
+            <FiArrowLeft onClick={() => navigate(-1)} />
+          </div>
+          <h2 className="text-sm md:text-lg  lg:text-2xl font-bold ml-4">Create Post</h2>
         </div>
-        <h2 className="text-sm md:text-lg  lg:text-2xl font-bold ml-4">Create Post</h2>
-      </div>
 
-      <form onSubmit={handleSubmit}>
-        <div className="mb-4">
-          <input
-            type="text"
-            name="title"
-            placeholder="Title"
-            className="w-full p-2 rounded-lg bg-gray-50 placeholder:text-gray-800 "
-            value={formData.title}
-            onChange={handleInputChange}
-          />
-        </div>
-        <div className="mb-4">
-          <input
-            type="text"
-            name="authorName"
-            placeholder="Author Name"
-            className="w-full p-2 rounded-lg bg-gray-50 placeholder:text-gray-800 "
-            value={formData.authorName}
-            onChange={handleInputChange}
-          />
-        </div>
-        <div className="mb-4">
-          <input
-            type="text"
-            name="imageLink"
-            placeholder="Image URL"
-            className="w-full p-2 rounded-lg bg-gray-50 placeholder:text-gray-800 "
-            value={formData.imageLink}
-            onChange={handleInputChange}
-          />
-        </div>
-        <div className="mt-4 mb-4 ">
-          <textarea
-            name="description"
-            placeholder="Description"
-            className="w-full p-2 rounded-lg bg-gray-50 placeholder:text-gray-800 "
-            value={formData.description}
-            onChange={handleInputChange}
-          />
-        </div>
+        <form onSubmit={handleSubmit}>
+          <div className="mb-4">
```

Comment:
Please check the formatting, though the files are not changed why are we seeing the changes